pkg/terminal/command.go

```diff
@@ -2849,17 +2848,65 @@ func printTracepoint(t *Term, th *api.Thread, bpname string, fn *api.Function, a
 	if t.conf.TraceShowTimestamp {
 		fmt.Fprintf(t.stdout, "%s ", time.Now().Format(time.RFC3339Nano))
 	}
+	wantindex := -1
+	rootindex := -1
+	stack := th.BreakpointInfo.Stacktrace
+	for i := range stack {
+		//fmt.Printf(" i=%d pc=%x fname=%s\n", i,stack[i].PC, stack[i].Function.Name())
+		curi := len(stack) - 1 - i
+		/*
+		   if stack[curi].Function.Name()=="main.main" {
+		           mainindex=curi
+		   }*/
+
+		if stack[curi].Function.Name() == th.Breakpoint.RootFuncName {
+			if rootindex == -1 {
+				rootindex = curi
+			}
+		}
+
+		if fn.Name() == stack[curi].Function.Name() {
+			//                        fmt.Printf(" i %d equal %s %s\n",curi,fn.Name(), stack[i].Function.Name())
+			wantindex = curi
+		}
+	}
+	//sdepth:=mainindex-wantindex
+	sdepth := rootindex - wantindex + 1
+	//fmt.Printf("follow calls depth %d\n",th.Breakpoint.TraceFollowCalls)
+	//fmt.Printf("Root func name %s\n",th.Breakpoint.RootFuncName)
```

Comment:
I don't understand what this is doing. Shouldn't you be counting all appearances of traced calls in the stack?